### ISSUE TITLE
Warning message in browser console.

### ISSUE THREAD

**Issue by Kappyh** [limonte/sweetalert2#733]
Hi
I'm using the sweet alert with angular version 5 and in browser console(Google Chrome) show this message:

`sweetalert2.all.js:129 SweetAlert2: The parameter "useRejections" is deprecated and will be removed in the next major release.`

My code:

```
swal({
                            title: 'Error title',
                            text: 'Thats a error budy',
                            type: 'error', 
                            confirmButtonText: 'Ok'
                          })
```

**Comment by limonte** [comment]
Thanks for reporting @Kappyh, this is the duplicate of #731, fixed in v7.0.3.

Update to latest in order to have that fix.

**Comment by Kappyh** [comment]
Thank you very much
